triggers/commands/help.ts

```diff
@@ -10,17 +12,15 @@ module.exports = <Command> {
     async execute(message, args) {
         
         if (args.length > 0) {
-            commands.forEach(command => {
-                if (command.name == args[0]) {
-                    const embed = new MessageEmbed()
-                        .setColor("#AC2B37")
-                        .setTitle(command.name)
-                        .setDescription(command.description)
-                        .addField("Aliases", `${command.aliases ? command.aliases : 'None'}`)
-                        .addField("Required Permissions", `${command.requiredPerms == 'public' ? 'None' : 'Restricted'}`)
-                    message.channel.send(embed)
-                }
-            })
+            let command = getCommand(args[0]);
```

Comment:
Something I just thought of: add a check for `command` coming back `undefined` (tell the user there are no commands that match)?

Reply:
good catch!